.github/workflows/cherryPick.yml

```diff
@@ -128,7 +128,7 @@ jobs:
         # Version: 2.4.3
         uses: repo-sync/pull-request@65194d8015be7624d231796ddee1cd52a5023cb3
         with:
-          source_branch: cherry-pick-staging-${{ github.event.inputs.PULL_REQUEST_NUMBER }}
+          source_branch: ${{ github.actor }}-cherry-pick-staging-${{ github.event.inputs.PULL_REQUEST_NUMBER }}
```

Comment:
Is there no way to get this or pass this along a different way?

Reply:
Maybe via the API to check for a workflow https://docs.github.com/en/rest/reference/actions#list-workflow-runs ?

Reply:
I agree this feels hacky, but the difficult part with using this API is identifying the workflow run that's associated with a given PR or version. Let me dig into it a bit.

Reply:
Basically, the information we have when we're running `markPullRequestsAsDeployed` is:

1. The version
2. The PR number

Then given those, we would need to find the `cherryPick.yml` workflow run that:

1. Generated that version, _OR_
1. Was executed with that PR number as input.

But unfortunately the inputs to a workflow run are not saved by their API, so I wasn't able to see how we could identify the correct workflow run.

Reply:
Open to suggestions if you see something I'm missing

Reply:
I don't see anything on the API docs on first read, but let me keep looking a bit further.